src/CMakeLists.txt

```diff
@@ -295,7 +298,14 @@ install(FILES
     ${CMAKE_BINARY_DIR}/exiv2lib_export.h
     DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/exiv2)
 
-install(EXPORT exiv2Config DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/exiv2")
+install(EXPORT exiv2Export 
+    DESTINATION "${CMAKE_INSTALL_DATADIR}/cmake/exiv2"
+    NAMESPACE exiv2::
```

Comment:
Adding the namespace is very important to take advantage of exported namespace targets, just like how you prefer to link to `ZLIB::ZLIB` compared to `ZLIB` due to the requirement the target exists in `target_link_libraries`.

Reply:
Any objections to making that namespace capitalised `Exiv2`, so e.g. the imported target becomes `Exiv2::exiv2`?